### ISSUE TITLE
Meta - accessibility issues

### ISSUE THREAD

**Issue by saghul** [jitsi/jitsi-meet#6090]
https://github.com/jitsi/jitsi-meet/issues/5992
https://github.com/jitsi/jitsi-meet/issues/5852
https://github.com/jitsi/jitsi-meet/issues/5817
https://github.com/jitsi/jitsi-meet/issues/5789
https://github.com/jitsi/jitsi-meet/issues/5739
https://github.com/jitsi/jitsi-meet/issues/5681
https://github.com/jitsi/jitsi-meet/issues/5308
https://github.com/jitsi/jitsi-meet/issues/3590
https://github.com/jitsi/jitsi-meet/issues/6089

**Comment by mgifford** [comment]
It seems like there should be an accessibility label that could be used to aggregate these issues in the future.  We know that there are going to be more issues.

**Comment by missmatsuko** [comment]
Should those issues be closed or not? I thought they should stay open until they're actually resolved.

**Comment by luixxiul** [comment]
I would agree with @missmatsuko; only duplicates should be closed.

**Comment by l3d00m** [comment]
Just discussed it, closing this issue as we used it basically as a label. This has now been replaced by adding a `accessibility` label to any issues in question. They have been reopened @missmatsuko. Sorry for any confusions caused!

**Comment by Echolon** [comment]
But let us keep this issue pinned, because people hopefully address existing issues first.

**Comment by l3d00m** [comment]
I disagree, nobody will look through a closed pinned issue that only contains the issue numbers. It's much easier to use the search for it. I'd also say that that logic could apply to any issue, not just accessibility, as they aren't really different.

**Comment by skolmer** [comment]
This one fixes a lot of accessibility issues related to outdated atlaskit components: https://github.com/jitsi/jitsi-meet/pull/8423

**Comment by skolmer** [comment]
This will fix the rest of the accessibility issues in jitsi-meet web https://github.com/jitsi/jitsi-meet/pull/8921
